src/ext/java/org/opentripplanner/ext/vectortiles/layers/vehicleparkings/StadtnaviVehicleParkingPropertyMapper.java

```diff
@@ -39,10 +39,12 @@ protected Collection<T2<String, Object>> map(VehicleParking vehicleParking) {
         new T2<>("realTimeData", vehicleParking.hasRealTimeData())
       )
     );
+    if (vehicleParking.getOpeningHours() != null) {
+      items.add(new T2<>("openingHours", vehicleParking.getOpeningHours().osmFormat()));
+    }
     items.addAll(mapI18NString("name", vehicleParking.getName()));
     items.addAll(mapI18NString("note", vehicleParking.getNote()));
     // TODO add when openingHours are implemented
```

Comment:
Update or remove this TODO

Reply:
Done. The fee hours are not used by the Stadtnavi frontend, so I'm removing that as well.